### PR TITLE
support nodejs client

### PR DESCRIPTION
**Pull request by shackbarth**
In a common nodejs use-case (using the `request` package) the `request.postRaw` is unparseable, and `request.post` is an object, not a string. These diffs are a bit inelegant but are a minimally invasive way to support this use-case.

I also wrote up some sample code for the readme.

### CODE REVIEW

**Comment by shackbarth** [comment]
It's a bit clever, but what the diffs do in plain English is:

If `request.post` is an object, and if the code _would have errored anyway_ (presumably but not necessarily using `request.postRaw`), then slot in a stringified version of the `request.post` object to be immediately unstringified again by the existing code.

**Comment by gvaartjes** [comment]
Thank you for taking your time to write the PR. 

I'm not sure about accepting it though. In your example you're sending the JSON as form POST, but the highcharts-convert.js script is designed to expect a JSON object in the request.body.

When you're posting the JSON to PhantomJS like in the snippet below, the highcharts-convert.js script is processing this without problems. 

**note:** You have to run stringify twice

```
var request = require('request'),
    infile = {
        xAxis: { categories: ["Jan", "Feb", "Mar"] },
        series: [{ data: [29.9, 71.5, 106.4] }]
    },
    data = { infile: JSON.stringify(infile)};

request.post({
  url:     'http://127.0.0.1:3003',
  json: true,
  headers: {
      "content-type": "application/json",
  },
  body: JSON.stringify(data)

}, function(error, response, body){
    console.log('done');
  console.log(body);
});
```

**Comment by shackbarth** [comment]
No problem. I'm happy the official solution already worked! My mistake was that I was passing the data in the `form` attribute instead of the `body` attribute. 

For what it's worth, for me (using `request` version 2.67.0) it was not necessary to stringify the second time.

**Comment by gvaartjes** [comment]
Nevertheless, thanks for your effort.
